parsers/AU.py

```diff
@@ -384,6 +391,19 @@ def fetch_production(country_code=None, session=None):
             print(row)
             continue
 
+        # Parse the datetime and check it matches the most recent one.
+        try:
+            plant_timestamp = arrow.get(row['Most Recent Output Time (AEST)']).datetime
+            # TODO: We should check it's not too old..
```

Comment:
This TODO comment should now be up where you read in latest_ts. But maybe just remove it altogether, I think it's fine.